src/Main.java

```diff
@@ -2567,7 +2769,11 @@ else if (region == 0x7) {
                     return inventory.contains(Items.JUMP_BOOTS);
                 }
                 else {
-                    return canLift(inventory);
+					/*
+					* Added MERCILESS logic for this check, which is Main Area - Lower Right.
+					* Using Ladder Scrolling at the start, this check can be reached without a Glove.
+					*/
+                    return (canLift(inventory) || difficulty >= difficulty.MERCILESS);
```

Comment:
`difficulty.MERCILESS` should be `Difficulty.MERCILESS`

Reply:
This will be fixed!